license-upgrade/apply-license.groovy

```diff
@@ -358,7 +354,9 @@ def productType() {
       }
     } else {
       def _plugin_oc_context = jenkins.model.Jenkins.instance.getPlugin('operations-center-context')
-      if(_plugin_oc_context != null && _plugin_oc_context.getWrapper().isActive()) {
+      def _plugin_folder_plus = jenkins.model.Jenkins.instance.getPlugin('cloudbees-folder-plus')
```

Comment:
@escoem I think the plugin id should be `cloudbees-folders-plus` as in
https://jenkins-updates.cloudbees.com/download/plugins/cloudbees-folders-plus/